pupil_src/player/vis_circle.py

```diff
@@ -42,9 +42,22 @@ def update(self,frame,events):
         else:
             thickness = self.thickness
 
-        pts = [denormalize(pt['norm_pos'],frame.img.shape[:-1][::-1],flip_y=True) for pt in events.get('gaze_positions',[])]
-        for pt in pts:
-            transparent_circle(frame.img, pt, radius=self.radius, color=(self.b, self.g, self.r, self.a), thickness=thickness)
+        pts = [(
```

Comment:
I think this should simply be a separate plugin.  Similar to viz_circle but that it works in fixations. Otherwise this one becomes kind of magic and non-intuitive.